### PR TITLE
Address issue #120 migration page in documentation

### PR DESCRIPTION
**Pull request by renee2468**
Signed-off-by: renee2468 <renee.albe@cybertec.at>

### CODE REVIEW

**Review by rcv-aws — APPROVED** [review]

File: _usage/migration.md [hunk 1]
```diff
@@ -6,73 +6,69 @@ nav_order: 3
...
-You can use SQL Studio Management Studio (SSMS) to export the DDL.
-Generate the script by clicking on your database in MS SQL Studio and use the
+### Reverse-engineer the SQL Server schema
+
+Once you have run an assessment, migrate your existing SQL Server schema to the new
+cluster. Generate the DDL for all SQL Server user databases which you want to migrate.
+The DDL is SQL code that describes database objects such
+as those that contain user data (for example, tables, indexes, and
+views) and user-written database code (stored procedures,
+user-defined functions, or triggers).
```
Comment (laurenz) [inline _usage/migration.md:42]: We don't need an explanation for DDL. I'd say this is well known.

File: _usage/migration.md [hunk 2]
```diff
@@ -6,73 +6,69 @@ nav_order: 3
...
-
-In case you are facing errors or problems, we recommend you check out our [discussion about
-differences in functionality](/docs/usage/limitations-of-babelfish).
+Make SQL code adjustments as needed. In case you are facing errors or problems,
+we recommend you check out our [discussion about differences in functionality](/docs/usage/limitations-of-babelfish).
+Run the DDL on your new Babelfish server to recreate your schema on Babelfish,
+either with sqlcmd or with SSMS. 
 
-In the next step you can move the data from one server to the other.
+In the next step you can migrate the data from one server to the other.
```
Comment (laurenz) [inline _usage/migration.md:58]: That suggests that this is described in the next section, which it isn't. Perhaps: "After that, you can migrate...".

File: _usage/migration.md [hunk 3]
```diff
@@ -86,10 +82,10 @@ to use the Babelfish TDS port.
 
 ### Performance considerations
 
-You should keep in mind that you have just left the universe of MS SQL and
-entered the world of PostgreSQL. This implies that performance won't be
-identical. In some cases, PostgreSQL will do better than MS SQL and in some
-situations, it will not perform as well, which is to be expected. 
+SQL might exhibit performance differences on different platforms, so performance
+adjustments may be necessary during a migration. You should keep in mind that you
+have just left the universe of Microsoft SQL Server and entered the world of PostgreSQL.
```
Comment (laurenz) [inline _usage/migration.md:87]: "Universe" is somewhat bombastic.

File: _usage/migration.md [hunk 4]
```diff
@@ -103,15 +99,15 @@ you can access them with the PostgreSQL protocol as well.  To do that, you can
 use any PostgreSQL client.  Make sure that you connect to the PostgreSQL port
 (by default 5432) and not to the TDS port (by default 1433).
 
-While table and column names will be identical, the names of the schemas in
-PostgreSQL will depend on the
-[migration mode](docs/installation/single-multiple):
+While object names will be identical, the names of the schemas in the migrated SQL
+Server user databases may look different when access through PG, depending on the
```
Comment (laurenz) [inline _usage/migration.md:103]: when access*ed* through *PostgreSQL*

File: _usage/migration.md [hunk 5]
```diff
@@ -103,15 +99,15 @@ you can access them with the PostgreSQL protocol as well.  To do that, you can
...
+[migration mode_GUC](docs/installation/single-multiple):
```
Comment (laurenz) [inline _usage/migration.md:104]: The underscore is in the wrong place, and "GUC" is internal PostgreSQL slang.
"Configuration parameter" would be better.

File: _usage/migration.md [hunk 6]
```diff
@@ -103,15 +99,15 @@ you can access them with the PostgreSQL protocol as well.  To do that, you can
...
 
-- in `single-db` migration mode, the schema names will be the same, no matter
-  if you connect via TDS or via the PostgreSQL protocol
+- in `single-db` migration mode, the schema names of the migrated SQL Server user
+databases will be the same when accessed using T-SQL or using PG SQL
```
Comment (laurenz) [inline _usage/migration.md:107]: "PG SQL" is ugly. Should be "the PostgreSQL protocol".
Also, there are two spaces missing - this is a continuation line.

File: _usage/migration.md [hunk 7]
```diff
@@ -103,15 +99,15 @@ you can access them with the PostgreSQL protocol as well.  To do that, you can
...
 
 - in `multi-db` migration mode, the schema names in PostgreSQL will be composed
   of the database name and the schema name that you used when connected
-  via the TDS protocol.  For example, if you created a table `t1` in schema
-  `s1` of database `d1` while connected via TDS, the table will be `d1_s1.t1`
-  in PostgreSQL.
+  through the TDS port.  For example, if you created a table `t1` in schema
+  `s1` of database `d1` while accessing using T-SQL, the table needs to be referenced
+  (or accessed) as d1_s1.t1 when connected to the PostgreSQL protocol.
```
Comment (laurenz) [inline _usage/migration.md:113]: Missing backticks: <code>&#96;d1_s1.t1&#96;</code>

**Review by laurenz — APPROVED** [review]
I have addressed my review comments in b1438428bc4a40ed26e35633507fb4f72e068035, so I'll mark this as "approved".

**Review by laurenz — APPROVED** [review]

**Review by rcv-aws — APPROVED** [review]
assuming all comments processsed, not double-checking

**Pull request merged**